diff --git a/index.php b/index.php
index fdf321e..9065dbe 100644
--- a/index.php
+++ b/index.php
@@ -100,7 +100,13 @@ $(function() {
         for($i = 0; $i<count($lines); $i++) {
             $line = $lines[$i];
             if($i%2===0) {
-                $line = str_replace(" : ", "", $line); // Remove : . Because %d or branch name only appears when branching, otherwise shows : instead of (branchName) :
+                if(strpos($line, " : ") === false) {
+                    $beginBranchName = strpos($line, "(");
+                    $endBranchName = strpos($line, ":")+1;
+                    $line = substr($line, 0, $beginBranchName) . "<span style='color:red;'>" . substr($line, $beginBranchName, $endBranchName-$beginBranchName) . "</span>" . substr($line, $endBranchName);
+                } else
+                    $line = str_replace(" : ", "", $line); // Remove : . Because %d or branch name only appears when branching, otherwise shows : instead of (branchName):
+
                 $pos = strpos($line, "*"); // ; sometimes the line starts with | *
                 $line = substr($line, 0, $pos) . "<span class='hover-notes' style='color:blue;'>" . substr($line, $pos) . "</span><br>";
                 echo $line;
